pid_gains:
  kP_linX: 150.0
  kI_linX: 30.0
  kD_linX: 0.0
  kP_omega: 40.0
  kI_omega: 8.0
  kD_omega: 0.0

feedforward:
  x0: 1500.0
  x1: 7.8
  a: 4.37
  b: 18.0

torque_limits:
  tq_Max: 280.0
  tq_Min: -280.0

velocity:
  max_velocity: 3.0
  min_velocity: 0.0

omega_limits:
  max_omega: 0.2
  min_omega: -0.2
  R_min: 2.0

deadband_velocity: 0.1

rate_limiter:
  dec_min_v: -0.1
  a_acc_v: 0.5
  b_acc_v: 3.0
  a_dec_v: -0.5
  b_dec_v: -15.0
  acc_max_v: 3.0
  dec_max_v: -3.0
  smoothing_factor_v: 20.0


  dec_min_w: -0.1
  a_acc_w: 0.5
  b_acc_w: 3.0
  a_dec_w: -0.5
  b_dec_w: -15.0
  acc_max_w: 1.0
  dec_max_w: -1.0
  smoothing_factor_w: 20.0

dt: 0.02

moving_velocity:
  v_moving_ss: 0.5
  v_moving: 0.4
  v_stopped: 0.01

